app/models/message.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: messages
+#
+#  id         :bigint           not null, primary key
+#  text       :text             not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  user_id    :bigint           not null
+#
+# Indexes
+#
+#  index_messages_on_user_id  (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+class Message < ApplicationRecord
+  validates :user_id, presence: true
```

Comment:
Please remove validates user_id. You can use optinal: true/false in association
https://guides.rubyonrails.org/association_basics.html

Reply:
`belongs_to` validated by default since Rails 5 IIRC (default required: true).

Reply:
`belongs_to :user, validate: true`?